worker.js

```diff
@@ -175,41 +157,36 @@ async function checkBalanceToProtect(balance) {
 }
 
 async function shieldTransaction(amount, shieldedAccountKeypair, recoveryAccount) {
-
     let attempt = 0;
-
-    //Create Simple Transaction
     let transaction = new solanaWeb3.Transaction();
-
     let recoveryAccountPK = new solanaWeb3.PublicKey(recoveryAccount);
 
-    // Add an instruction to execute
     transaction.add(solanaWeb3.SystemProgram.transfer({
         fromPubkey: shieldedAccountKeypair.publicKey,
         toPubkey: recoveryAccountPK,
         lamports: amount,
     }));
-    
+
     try {
         attempt++;
         let result = await solanaWeb3.sendAndConfirmTransaction(connection, transaction, [shieldedAccountKeypair])
-        console.log('Shielded %d SOL', amount / 1000000000 );
+        console.log('Shielded %d SOL', amount / 1000000000);
         console.log('Transaction ID: %s', result);
         console.log('SOL balance is now 0. Suck it hackers.');
-        log('log', 'Shielded ' + amount / 1000000000 + 'SOL.');
```

Comment:
@whymidnight This is for further implementation around websockets and displaying info to user on the site. It essentially acts as a dual function to  console.log  and send to the websocket. I can reimplement it though if I can figure out how to get a working implementation that works on production, though.